src/Security/Security.php

```diff
@@ -397,18 +460,15 @@ public function LoginForm()
      * @return array Returns an array of available login forms (array of Form
      *               objects).
      *
-     * @todo Check how to activate/deactivate authentication methods
      */
-    public function GetLoginForms()
+    public function getLoginForms()
```

Comment:
Shouldn't this be

return $authenticator->getloginHandler($this->Link())->loginForm()

?

This method is ment to return a list of forms, not a list of HTTPResponses
Firesphere 2 days ago Contributor

It seems to be not even called anywhere anymore...
tractorcow a day ago Contributor

There are login forms that have multiple login methods (e.g. oauth / member login). It's ment to display multiple forms, grouped with tabs.